src/Microsoft.ML.TimeSeries/ExtensionsCatalog.cs

```diff
@@ -156,7 +156,7 @@ public static class TimeSeriesCatalog
         /// <param name="outputColumnName">Name of the column resulting from data processing of <paramref name="inputColumnName"/>.
         /// The column data is a vector of <see cref="System.Double"/>. The length of this vector varies depending on <paramref name="detectMode"/>.</param>
         /// <param name="inputColumnName">Name of column to process. The column data must be <see cref="System.Double"/>.</param>
-        /// <param name="threshold">The threshold to determine anomaly, score larger than the threshold is considered as anomaly. Must be in [0,1]. Default value is 0.3.</param>
+        /// <param name="threshold">The threshold to determine an anomaly. An anomaly is detected when the calculated anomaly score for a given time-series chunk is more than the set threshold. This threshold must fall between [0,1], and its default value is 0.3.</param>
```

Comment:
>/// <param name="threshold">The threshold to determine an anomaly. An anomaly is detected when the calculated anomaly score for a given time-series chunk is more than the set threshold. This threshold must fall between [0,1], and its default value is 0.3.</param> [](start = 8, length = 263)

The "score" here refers to "raw score", this is the score output by SR for each point. And there is an "anomaly score" under AnomalyAndMargin mode, this is a score calculated according to user's sensitivity setting when a point is detected as an anomaly by SR. So I would suggest this line to be:
/// <param name="threshold">The threshold to determine an anomaly. An anomaly is detected when the calculated SR raw score for a given point is more than the set threshold. This threshold must  fall between [0,1], and its default value is 0.3.</param>